src/Native/gc/sample/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 include_directories(..)
 include_directories(../env)
 
+add_definitions(-DNO_STRESS_LOG)
```

Comment:
Does this need to be defined for the sample project? As far as I can tell, it is only used in the runtime not it in the GC.